src/dev/env.md

```diff
@@ -19,6 +19,15 @@ The following stages are supported:
  - `jit-post-opt`: the IR for the trace after LLVM has optimised it. This is
    the IR that will be submitted to the LLVM code generator.
 
+### `YKD_SERIALISE_COMPILATION`
+
+If `YKD_SERIALISE_COMPILATION=1`, when an interpreter thread calls the the
```

Comment:
This sentence doesn't really parse. Rephrase?

Reply:
Does 593b49c help?